op-mode-definitions/show-conntrack.xml.in

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="conntrack">
+        <properties>
+          <help>Show conntrack entries in the conntrack table</help>
+        </properties>
+        <children>
+          <node name="table">
+            <properties>
+              <help>Show conntrack entries for table</help>
+            </properties>
+            <command>netstat -an</command>
```

Comment:
If we are about to introduce machine-readable op mode, we should not use new direct calls, it should be a part of the script.

Reply:
This command is not required.
Fixed.